src/main/java/com/codeborne/selenide/junit5/BrowserStrategyExtension.java

```diff
@@ -0,0 +1,17 @@
+package com.codeborne.selenide.junit5;
+
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import static com.codeborne.selenide.WebDriverRunner.closeWebDriver;
+
+/**
+ * @author Aliaksandr Rasolka
+ * @since 4.12.2
+ */
+public class BrowserStrategyExtension implements AfterAllCallback {
```

Comment:
1. Add javadoc with usage sample. 
2. It's not clear from javadoc, is this "browser per class" or "browser per method"?